### DISCUSSION TITLE
Add font-stack for headings

### DISCUSSION THREAD

**Discussion by sscherfke** [pradyunsg/furo#431] [Ideas] [answered]
It would be nice if there was a `font-stack--headings` for change the fonts of `<h*>` elements and titles.  This would allow users to add *some* branding to the fonts without sacrificing readability of the main content.  The default could be the same value as `font-stack` has.

**Comment by pradyunsg** [comment, marked as answer]
This is a nice idea! Done: eb9d7496a8a81aa8c8662737c170e2156806ec40
